integrations/castle/lib/index.js

```diff
@@ -5,39 +5,44 @@
  */
 
 var integration = require('@segment/analytics.js-integration');
+var objCase = require('obj-case');
+var newDate = require('new-date');
+var isoDate = require('@segment/isodate');
+var toIsoString = require('@segment/to-iso-string');
+var isObject = require('isobject');
 
 /**
  * Expose `Castle` integration.
  */
 
-var Castle = (module.exports = integration('Castle')
+var CastleIntegration = (module.exports = integration('Castle')
+  .global('CastleSegment')
+  .global('casData')
   .option('publishableKey', '')
-  .option('autoPageview', false)
-  .option('cookieDomain', false)
-  .tag('<script src="//d2t77mnxyo7adj.cloudfront.net/v1/cs.js">'));
+  .option('cookieDomain', '')
```

Comment:
Awesome, looks like this already shows up as a string in the destination settings page so this change is great!